std-bits/base/src/main/java/module-info.java

```diff
@@ -1,5 +1,13 @@
+import org.enso.base.enso_cloud.DataLinkSPI;
+import org.enso.base.enso_cloud.EnsoFileDataLinkSPI;
 import org.enso.base.enso_cloud.EnsoPathFileSystemSPI;
+import org.enso.base.file_format.ByteFormatSPI;
+import org.enso.base.file_format.FileFormatSPI;
+import org.enso.base.file_format.JSONFormatSPI;
+import org.enso.base.file_format.TextFormatSPI;
+import org.enso.base.file_format.XMLFormatSPI;
 import org.enso.base.file_system.FileSystemSPI;
```

Comment:
@radeusgd, there is a small _terminology issue_ and encapsulation/security issue. [SPI stands for service provider interface](http://wiki.apidesign.org/wiki/SPI) - e.g. interface towards providers of some service. `FileFormatSPI` is such an interface. However `ByteFormatSPI`, `TextFormatSPI`, etc. _are not provider interfaces_!

Those are implementations. Thus adding `SPI` suffix to them isn't correct. I'd give them `Impl` suffix. Moreover such _implementations shouldn't be in a exported package_. Modules should export only an absolute minimum to [lower the conceptual surface](http://wiki.apidesign.org/wiki/Conceptual_surface) of the API. 

Some refactoring is desirable.